assets/js/modules/ads/utils/validation.js

```diff
@@ -49,7 +49,7 @@ export function isValidPaxConversionID( value ) {
 /**
  * Checks if the given extCustomerID ID is valid.
  *
- * @since 1.126.0
+ * @since n.e.x.t
```

Comment:
This one is also not needed, right?

Reply:
Correct, I got confused with the removal and re-addition, this is now sorted to be 1.126.0 again